src/platform_impl/linux/x11/window.rs

```diff
@@ -1449,8 +1449,37 @@ impl UnownedWindow {
     }
 
     #[inline]
-    pub fn set_cursor_hittest(&self, _hittest: bool) -> Result<(), ExternalError> {
-        Err(ExternalError::NotSupported(NotSupportedError::new()))
+    pub fn set_cursor_hittest(&self, hittest: bool) -> Result<(), ExternalError> {
+        let shape_input = 2;
+        let mut rectangles: Vec<ffi::XRectangle> = Vec::new();
+        if hittest {
+            let size = self.inner_size();
+            rectangles.push(ffi::XRectangle {
+                x: 0,
+                y: 0,
+                width: size.width as u16,
+                height: size.height as u16,
+            })
+        }
+        unsafe {
+            let region = (self.xconn.xfixes.XFixesCreateRegion)(
+                self.xconn.display,
+                rectangles.as_mut_ptr(),
+                rectangles.len() as i32,
+            );
```

Comment:
Can this region be cached?

Reply:
Where? It has the window size in it. Also at least in my use case, I'm usually just disabling hittest once and leaving it disabled for any given window.